src/webui/www/private/scripts/mocha-init.js

```diff
@@ -957,6 +958,26 @@ const initializeWindows = function() {
         return torrentsTable.selectedRowsIds().join("\n");
     };
 
+    exportTorrentFN = function() {
+        const hashes = torrentsTable.selectedRowsIds();
+        for (const hash of hashes) {
+            const row = torrentsTable.rows.get(hash);
+            if (!row) return
+
+            const name = row.full_data.name;
+            const url = new URI("api/v2/torrents/export");
+            url.setData("hash", hash);
+
+            // download response to file
+            const element = document.createElement("a");
+            element.setAttribute("href", url);
+            element.setAttribute("download", name + ".torrent");
+            document.body.appendChild(element);
+            element.click();
```

Comment:
Is there really no direct way to do this without creating some element behind the scenes and emulating the impact on it?

Reply:
I initially looked into sending the `Content-Disposition` header, which indicates to the browser that the response should be downloaded, but it doesn't appear that there's plumbing yet to allow setting arbitrary headers from API routes.